kss-config.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Do we need to have this top level?

Reply:
I would move it inside the folder with all the kss files.

Reply:
yes because of `package.json`